src/modules/apis.js

```diff
@@ -0,0 +1,132 @@
+const baseUrl = 'https://api.spacexdata.com/v3/rockets';
+const likeBaseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/5GbgWWZPMVcYokoIs4ID/likes';
+const commentBaseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps';
+
+const appId = '5GbgWWZPMVcYokoIs4ID';
+
+// get all rockets 🚀🚀
+const getAllRockets = async () => {
+  try {
+    const response = await fetch(`${baseUrl}`, {
+      method: 'GET',
+      headers: { 'Content-Type': 'application/json' },
+    });
+    if (!response.ok) {
+      throw new Error('Could not fetch data');
+    }
+    const responseData = await response.json();
+
+    // the responese data is an array so loop through and extract the relevant data needed
+    const rocketsData = responseData.map(({
+      /* eslint-disable camelcase */
+      rocket_name, rocket_id, flickr_images, cost_per_launch,
+      success_rate_pct, mass, rocket_type,
+    }) => ({
+      rocket_type,
+      mass,
+      success_rate_pct,
+      cost_per_launch,
+      rocket_name,
+      rocket_id,
+      flickr_images: flickr_images[0], // Extract the first element from the flickr_images array
+    }));
```

Comment:
- [ ] You did a good job working on this milestone! 💯 However, please avoid disabling linter rules as they serve as a way to provide a consistent code format and ensure you use best practices. Instead, consider fixing the errors! 🤓 🔧 

In this case, I suggest you use `camelCase` to name the properties of the returning object:


```js
const rocketsData = responseData.map(({/* ...properties */})=>({
  type: rocket_type,
  mass,
  successRatePct: success_rate_pct,
  /* ... */
}))
```

Reply:
Please did you really check the code to see I'm destructuring data directly from the endpoint. How can i mutate the variable name. So it's necessary I disable the camel case

Reply:
resolved